Example/PrebidDemoJava/src/main/java/org/prebid/mobile/app/CustomApplication.java

```diff
@@ -44,8 +44,6 @@ public void onCreate() {
 
         PrebidMobile.setShareGeoLocation(true);
         PrebidMobile.setApplicationContext(getApplicationContext());
-        WebView obj = new WebView(this);
-        obj.clearCache(true);
```

Comment:
why did you remove these lines ?

Reply:
Why do we have this?

Reply:
I don't know
You added it as a part of commit
c3b0a76b222894f7941f78cb9102cbead724137d

Usually I use this technic to review all requests and responses(no caching) using Charles

Reply:
I don't remember why I added this, it seems not meaningful to the demo app.
Do you approve the removing?